Classes/Issues/Merge/GithubClient+Merge.swift

```diff
@@ -18,7 +18,7 @@ extension GithubClient {
         repo: String,
         number: Int,
         type: IssueMergeType,
-        error: @escaping () -> Void
+        completionHandler: @escaping (_ success: Bool) -> Void
```

Comment:
Why do you ignore it still? That wouldn't help to understand it at the call site 😄

Reply:
@BasThomas functional type can't have a argument label. If you ask about it.

Reply:
Ah.